cloudnet-launcher/src/main/java/de/dytanic/cloudnet/launcher/CloudNetLauncher.java

```diff
@@ -311,11 +312,11 @@ private static void startApplication(List<String> args, File targetDirectory, Co
         dependencyResources.add(target.toURI().toURL());
         dependencyResources.add(driverTarget.toURI().toURL());
 
-        ClassLoader classLoader = new RuntimeClassLoader(dependencyResources.toArray(new URL[0]), ClassLoader.getSystemClassLoader());
+        ClassLoader classLoader = new URLClassLoader(dependencyResources.toArray(new URL[0]));
+
         prepareApplication(variables);
 
         Method method = classLoader.loadClass(mainClazz).getMethod("main", String[].class);
-        method.setAccessible(true);
```

Comment:
Why removing this?

Reply:
main method have to be accessible by default, this is unnecessary